CMakeLists.txt

```diff
@@ -61,11 +70,35 @@ install(FILES
   src/binode.h
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/keyfinder")
 
-set(LIBKEYFINDER_VERSION 2.2.3)
+# pkgconfig
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libkeyfinder.pc.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/libkeyfinder.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libkeyfinder.pc DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 
+# CMake config
+set(KEYFINDER_INSTALL_CMAKEDIR "lib/cmake/KeyFinder")
```

Comment:
```suggestion
set(KEYFINDER_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/KeyFinder")
```

Reply:
Regarding case: https://github.com/mixxxdj/libkeyfinder/pull/7#discussion_r557599881

Unsure about `lib` vs `CMAKE_INSTALL_LIBDIR`. On my system they are the same, but I think it can also be `lib64`. Are the cmake files in `/usr/lib/cmake` or `/usr/lib64/cmake` on those systems? The tutorials just hardcode it:
- https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#id6
- https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#example-generating-package-files